src/parser/jobs/gnb/index.tsx

```diff
@@ -4,21 +4,26 @@ import CONTRIBUTORS, {ROLES} from 'data/CONTRIBUTORS'
 import {Meta} from 'parser/core/Meta'
 import {changelog} from './changelog'
 
-const description = t('gnb.about.description')`
-You can write the description for the job here. It supports a superset of
-[markdown](https://commonmark.org/).
+const description = t('gnb.about.description')`This analyzer looks for the low-hanging, easy to spot issues in your gameplay that can be fixed to improve your damage across a fight as Gunbreaker.
+If you're looking to learn about how exactly the job plays and functions from the ground up, take a look at a few basic guides:
+
+TODO: FILL THIS OUT WITH LINKS
+
+If you have any suggestions about the module, feel free to ping me on Discord at Lhea#7581.
 `
 
 export default new Meta({
 	modules: () => import('./modules' /* webpackChunkName: "jobs-gnb" */),
 
 	Description: () => <TransMarkdown source={description}/>,
-	// supportedPatches: {
-	// 	from: '✖',
-	// 	to: '✖',
-	// }
+
+	supportedPatches: {
```

Comment:
Probably isn't quite ready to be marked as supported - leaving this commented out will still let people use the job, but it'll be quite explicitly marked as not ready.

If you're happy with the existing logic and feel it _is_ ready to be marked as supported, should clean up the description so we're not dropping TODOs on users for a supported job.

Reply:
Fixed this up until we're ready to mark it as supported. Also updated the description.